packages/create-astro/test/project-name.test.js

```diff
@@ -9,25 +9,22 @@ describe('project name', async () => {
 	it('pass in name', async () => {
 		const context = { projectName: '', cwd: './foo/bar/baz', prompt: () => {} };
 		await projectName(context);
-
-		expect(context.cwd).to.eq('./foo/bar/baz');
-		expect(context.projectName).to.eq('baz');
+		assert.strictEqual(context.cwd, './foo/bar/baz');
```

Comment:
`assert.equal` and `assert.strictEqual` are the same because we import `node:assert/strict`. Can we use `assert.equal` like we are doing in the rest of the code base?

Reply:
hey, yeah you're right, `.equal` will be better as it maintains the consistancy through out the codebase. Give me a minute I'll update this.